component/slos.libsonnet

```diff
@@ -7,7 +7,7 @@ local kube = import 'lib/kube.libjsonnet';
 local inv = kap.inventory();
 // The hiera parameters for the component
 local params = inv.parameters.appcat;
-
+local alertsDisabled = if inv.parameters.appcat.slos.alertsEnabled then false else true;
```

Comment:
please rename this variable to alertsEnabled so we have some order :D

Reply:
yeah I guess I exagerated here :D

Reply:
Actually I simplified the variable, now should be good

Reply:
looks better, but naming is still weird, in class You use alertsEnabled in jsonnet alertsDisabled and You inverse it - why?

Reply:
giving it 3rd look - why do You even declare new variable and just not use param directly?